### ISSUE TITLE
Output mime type action feedback

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#93955]
Refs: #93742

Some feedback regarding the Output mime type action

1. Use a hover message
2. Make it accessible, so users can navigate to it via keyboard - or have it in command palette
3. Do not trigger on right click, only trigger on left click
4. "bulitin"  description of each quick pick option not that useful. Also not readable. I suggest to not use it, or if used consider changing to "built-in"

![Screenshot 2020-03-31 at 16 24 05](https://user-images.githubusercontent.com/1926584/78037754-3d47cf80-736c-11ea-8aba-35b477c9093f.png)

**Comment by rebornix** [comment]
Now users can navigate through tabbing.

![Kapture 2020-04-07 at 17 06 22](https://user-images.githubusercontent.com/876920/78730621-3d0e7d80-78f2-11ea-8290-f75a6b297898.gif)

**Comment by isidorn** [comment]
@rebornix nice, thanks! Should I open new issues for my concerns 1., 3. and 4?

**Comment by rebornix** [comment]
1 and 4 are fixed as well. I'm a bit not clear about 2, doesn't that mean "Tab"? If not feel free to create a new issue with details.

**Comment by isidorn** [comment]
Yeah 2 means accessible via Tab and that works. That is why I think that one is fine and I did not mention it in my comment. I was only conerned for 1 3 and 4
